app/src/main/java/net/vonforst/evmap/storage/ChargeLocationsDao.kt

```diff
@@ -218,6 +228,16 @@ class ChargeLocationsRepository(
         return CacheLiveData(dbResult, apiResult)
     }
 
+    /**
+     * Numeric date for database limit required limit on some APIs
+     */
+    public fun afterDate(dataSource: String): Long {
+        val after = Calendar.getInstance()
+        var days = this.api.value!!.keepDays
```

Comment:
use `val`, not `var` if the variable doesn't need to be mutable

Reply:
Ok